src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
This file shouldn't be included in this pull request.

Reply:
Yes but it is not relevant to the subject of your PR. This is only about the teleporter.